### PR TITLE
Release - host on cloudflare, dep updates

### PR DESCRIPTION
**Pull request by hobbitronics**

### CODE REVIEW

File: src/data/analytics.js [hunk 1]
```diff
@@ -1,21 +1,24 @@
+// TODO: rename this in cloudflare
```
Comment (briskt) [inline src/data/analytics.js:1]: Remember this TODO. Do you have a new value for it?

**Review by briskt — COMMENTED** [review]
I would have preferred to make the Google Analytics change later so this can move forward, but if you can finish that out we can press on.

File: src/data/analytics.js (hunk 1 above)
Comment (hobbitronics) [reply src/data/analytics.js]: Not sure who has access to our gtag id so I've changed it back to analytics for now.

Comment (briskt) [reply src/data/analytics.js]: I think a new one will need to be created, and any of us can probably do it. But it's up to @devon-sil to decide who should set it up, I think.

**Review by briskt — APPROVED** [review]

**Pull request merged**